### PR TITLE
Switch to .NET Standard 2.0 (as discussed in issue #282)

### PR DESCRIPTION
**Pull request by rdeago**

### CODE REVIEW

**Comment by rdeago** [comment]
Please @geoperez do not create a new release yet. Next PR will close #282 for real.

File: src/Unosquare.Labs.EmbedIO/Modules/StaticFilesModule.cs [hunk 1]
```diff
@@ -36,8 +36,7 @@ public class StaticFilesModule : FileModuleBase, IDisposable
 
         private readonly VirtualPathManager _virtualPathManager;
 
-        private readonly ConcurrentDictionary<string, Tuple<long, string>> _fileHashCache =
-            new ConcurrentDictionary<string, Tuple<long, string>>();
+        private readonly ConcurrentDictionary<string, (long DateTicks, string HashCode)> _fileHashCache = new ConcurrentDictionary<string, (long, string)>();
```
[the remark this answers is not in the document]
Comment (rdeago) [reply src/Unosquare.Labs.EmbedIO/Modules/StaticFilesModule.cs]: Apparently Codacy is not `ValueTuple`-aware.

File: src/Unosquare.Labs.EmbedIO/Modules/StaticFilesModule.cs [hunk 2]
```diff
@@ -429,11 +424,11 @@ private Stream GetFileStream(IHttpContext context, FileSystemInfo fileInfo, bool
...
         {
             var currentHash = _fileHashCache.TryGetValue(localPath, out var currentTuple) &&
-                              fileDate.Ticks == currentTuple.Item1
-                ? currentTuple.Item2
+                              fileDate.Ticks == currentTuple.DateTicks
+                ? currentTuple.HashCode
                 : $"{buffer.ComputeMD5().ToUpperHex()}-{fileDate.Ticks}";
 
-            _fileHashCache.TryAdd(localPath, new Tuple<long, string>(fileDate.Ticks, currentHash));
+            _fileHashCache.TryAdd(localPath, (fileDate.Ticks, currentHash));
```
[the remark this answers is not in the document]
Comment (rdeago) [reply src/Unosquare.Labs.EmbedIO/Modules/StaticFilesModule.cs]: Apparently Codacy is not `ValueTuple`-aware.

**Comment by geoperez** [comment]
Thanks again! Great work!

**Pull request merged**